fightScreen.py

```diff
@@ -0,0 +1,633 @@
+import sys
+from random import choice as randChoice
+from time import sleep
+from PySide6.QtCore import QSize, Qt, QTimer
+from PySide6.QtGui import QPixmap, QMovie, QColor
+from PySide6.QtWidgets import (
+    QApplication,
+    QMainWindow,
+    QLabel,
+    QWidget,
+    QGridLayout,
+    QSpacerItem,
+    QSizePolicy,
+    QPushButton,
+)
+from meter import Meter
+from arsenal import Arsenal
+
+
+class MainWindow(QMainWindow):
+    def __init__(self):
+        super().__init__()
+
+        self.imageDir = "C:/classGame/"
```

Comment:
Absolute pathing is going to be a nightmare for us (especially because I'm on Linux). Change this to a relative path `./assets` and put the images in there.

Reply:
Done.